DxOper.pas

```diff
@@ -141,7 +144,7 @@ function TDxOperator.IsMyCall: TCallCheckResult;
 
       D := M[x-1,y-1];
       //'?' matches any char
-      if not (C[x] in [C0[y], '?']) then Inc(D, W_D);
+      if not CharInSet(C[x], [C0[y], '?']) then Inc(D, W_D); //grg1

```

Comment:
Delphi 10 creates warning for "in", warning suggests usage of CharInSet instead.
So I replaced "in" with CharInSet() in all places, not just here